web/src/main/resources/static/js/tree.js

```diff
@@ -339,9 +363,14 @@ var tree = (function () {
                     '<br/>';
                 _qtipContent += '<b>Name:</b> ' + d.name.replace(/\(\w+\)/gi, '') + '<br/>';
                 _qtipContent += '<b>Main type:</b> ' + d.mainType + '<br/>';
-// Temporarily removed until we properly fetch nci/umls values
-//                _qtipContent += '<b>NCI:</b> ' + nci_link  + '<br/>';
-//                _qtipContent += '<b>UMLS:</b> ' + umls_link  + '<br/>';
+                _qtipContent += '<b>NCI:</b> ' + nci_links.join(",") + '<br/>';
+                _qtipContent += '<b>UMLS:</b> ' + umls_links.join(",") + '<br/>';
+                //$.each(nci_links, function ( index, value ) {
```

Comment:
The commented out code should definitely not be there.  That was definitely a mistake.